### DISCUSSION TITLE
if a vnet has two ip ranges reserved at diff times then how to apply 2x tags?

### DISCUSSION THREAD

**Discussion by omerzubair** [Azure/ipam#216] [General]
I was unable to assign the tag with same name and diff value on the same vnet. 
how to assign 2x CIDR reservation to same vnet?

![image](https://github.com/Azure/ipam/assets/7728453/a8d56efa-b328-4b9e-8476-aa5d47e4d5ab)

**Comment by DCMattyG** [comment]
Hi @omerzubair, you can simply put multiple tag values in a comma separated format.

Example:
```
X-IPAM-RES-ID: xxxxxxxxxxxxxxxx, yyyyyyyyyyyyyyyy
```

Hope that helps!

**Reply by omerzubair** [reply]
i will give this a try and revert!

**Reply by omerzubair** [reply]
![image](https://github.com/Azure/ipam/assets/7728453/4d779960-33a9-437b-94eb-ab08080b9d33)
![image](https://github.com/Azure/ipam/assets/7728453/329f0d01-dd82-436d-9723-e8f8c2cb5edb)
![image](https://github.com/Azure/ipam/assets/7728453/36767a67-be79-4cfc-92b6-d86d0b68db93)

SUPER!!! it picked up in no time in GUI as well!
